services/server/config/config_test.go

```diff
@@ -31,8 +31,8 @@ func TestMergeConfigs(t *testing.T) {
 	a := &Config{
 		Version:          2,
 		Root:             "old_root",
-		RequiredPlugins:  []string{"old_plugin"},
-		DisabledPlugins:  []string{"old_plugin"},
+		RequiredPlugins:  []string{"io.containerd.old_plugin.v1"},
+		DisabledPlugins:  []string{"io.containerd.old_plugin.v1"},
```

Comment:
The existing test should pass without modification ?

Reply:
Ah, yes, I wanted to post a comment about that, but looks like I forgot.

It's not needed indeed to make the set pass; reason I made the change was to make the test values better match reality (as we don't allow the old values to be used).

Happy to revert that change if you prefer not to update them though (let me know)